file-events/src/file-events/headers/win_fsnotifier.h

```diff
@@ -59,22 +60,27 @@ enum class WatchPointStatus {
 
 class WatchPoint {
 public:
-    WatchPoint(Server* server, size_t bufferSize, const u16string& path);
+    WatchPoint(Server* server, size_t eventBufferSize, const wstring& path);
```

Comment:
I saw that you converted the string we hold to `wstring`s. What is the advantage of this?

Reply:
With the new changes we are converting back and forth between long/short paths a bunch more, and doing all that arithmetic in `wstring` land means we don't have to also convert back-and-forth between `u16string` and `wstring` so much.